get_images.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
+
+"""
+    images.py
+
+    MediaWiki Action API Code Samples
+    Demo of `Images` module: Sending GET request to images
+    MIT license
+"""
+# Note: This code relies on the third party library, requests. 
```

Comment:
Nice that you've added this statement here, but may be let's stick to our hello world template [here](https://www.mediawiki.org/wiki/API:Documentation_template#Sample_code) :-)

Reply:
Removed from renamed file